pkgs/development/python-modules/blivet/default.nix

```diff
@@ -0,0 +1,109 @@
+{
+  lib,
+  pkgs,
+  python,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pygobject3,
+  libblockdev,
+  bytesize,
+  pyudev,
+  dbus-python,
+  util-linux,
+  kmod,
+  libndctl,
+  nvme-cli,
+  dosfstools,
+  e2fsprogs,
+  hfsprogs,
+  xfsprogs,
+  f2fs-tools,
+  ntfs3g,
+  btrfs-progs,
+  reiserfsprogs,
+  mdadm,
+  lvm2,
+  gfs2-utils,
+  cryptsetup,
+  multipath-tools,
+  dracut,
+  stratisd,
+}:
+
+let
+  libblockdevPython = (libblockdev.override { python3 = python; }).python;
+in
+buildPythonPackage rec {
+  pname = "blivet";
+  version = "3.10.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "storaged-project";
+    repo = pname;
+    rev = "${pname}-${version}";
```

Comment:
```suggestion
    repo = "blivet";
    rev = "blivet-${version}";
```

Reply:
I am looking at other definitions in nixpkgs and it seems that most use `${pname}` in this circumstance. Would you mind explaining why repeating the package name here is an improvement even though it doesn't seem to be common practice currently?

Reply:
Because a change in `pname` should not change anything in `src`. Even if you change `pname` you still want to fetch the same repo and use the same tag name structure. Here's an issue about that https://github.com/nix-community/nixpkgs-lint/issues/21

Reply:
Thank you!